index.html

```diff
@@ -95,6 +95,13 @@
 
        @media (max-width: 979px) {
```

Comment:
Really great work @vedikaag99 ,

One suggestion can we put the media queries at the last, I think that would increase the visibility

Reply:
@ArunTeltia done. please check now.